Packs/Fortimail/Integrations/Fortimail/Fortimail.yml

```diff
@@ -0,0 +1,2214 @@
+      type: String
+    - contextPath: FortiMail.RecipientPolicy.auth_allow_smtp
+      description: Whether to authenticate SMTP connections using the authentication profile configured in sensitive-data.
+      type: String
+    - contextPath: FortiMail.RecipientPolicy.smtp_diff_identity
+      description: Whether to reject different SMTP sender identity for authenticated user.
+      type: String
+    - contextPath: FortiMail.RecipientPolicy.smtp_diff_identity_ldap
+      description: Whether to verify SMTP sender identity with LDAP for authenticated email.
+      type: String
+    - contextPath: FortiMail.RecipientPolicy.smtp_diff_identity_ldap_profile
+      description: LDAP profile for SMTP sender identity verification.
+      type: String
+    - contextPath: FortiMail.RecipientPolicy.certificate_required
+      description: Whether to allow email users to log in to their per-recipient spam quarantine by presenting a certificate rather than a user name and password.
+      type: String
+  - name: fortimail-tls-profile-list
+    description: List TLS profiles. TLS profiles allow you to selectively disable or enable TLS for specific email recipient patterns, IP subnets, and so on. A common use of TLS profiles is to enforce TLS transport to a specific domain and verify the certificate of the receiving servers. if an name is given, the command will return the information about the wanted TLS profile. Mainly used in the configuration of Access control rule.
+    arguments:
+    - name: name
+      description: The name of the TLS profile.
+      default: true
+    - name: all_results
+      description: Whether to retrieve all results by overriding the default limit.
+      defaultValue: 'false'
+      auto: PREDEFINED
+      predefined:
+      - 'true'
+      - 'false'
+    - name: limit
+      description: The maximum number of records to retrieve.
+      defaultValue: '50'
+    outputs:
+    - contextPath: FortiMail.TLSprofile.mkey
+      description: The name of the TLS profile.
+      type: String
+    - contextPath: FortiMail.TLSprofile.level
+      description: The TLS level.
+      type: Number
+    - contextPath: FortiMail.TLSprofile.action
+      description: The TLS action on failure.
+      type: String
+    - contextPath: FortiMail.TLSprofile.comment
+      description: A brief comment for the TLS profile.
+      type: String
+    - contextPath: FortiMail.TLSprofile.is_referenced
+      description: Number of referencing entities.
+      type: Number
+  - name: fortimail-ldap-group-list
+    description: List LDAP profiles. LDAP groups lets to allow match email addresses as sender or recipients with the LDAP profile authentication in the Access control rule configuration and is the authentication profile in case the authentication type in IP policy is LDAP. if an name is given, the command will return the information about the wanted LDAP profile. Mainly used in the configuration of Access control rule and the IP policy.
```

Comment:
```suggestion
    description: List LDAP profiles. LDAP groups enables matching email addresses as senders or recipients with the LDAP profile authentication in the access control rule configuration and is the authentication profile in case the authentication type in IP policy is LDAP. If a name is given, the command will return the information about the specified LDAP profile. Mainly used in the configuration of access control rule and the IP policy.
```